helmfile/charts/grafana-dashboards/dashboards/uptime-dashboard.json

```
@@ -901,7 +906,37 @@
   "style": "dark",
   "tags": [],
   "templating": {
-    "list": []
+    "list": [
+      {
+        "current": {
+          "selected": true,
+          "text": [
+            "user-api-server"
+          ],
+          "value": [
+            "user-api-server"
+          ]
+        },
+        "cacheTimeout": null,
+        "datasource": "prometheus-sc",
+        "definition": "label_values(probe_success{instance=~\".*kube-apiserver.*\"}, target)",
+        "hide": 0,
+        "includeAll": false,
```

```suggestion
        "includeAll": true,
```
And possibly have it default to all.

---

Changed it so default is now "all"